src/client/src/reducers/azureLoginReducers/subscriptionDataReducer.ts

```diff
@@ -5,19 +5,22 @@ import AzureActionType from "../../actions/azureActions/azureActionType";
 {
     SubscriptionData: {
```

Comment:
should this just be removed?

Reply:
This just shows what the store would look like, and we have it in all the reducers so I think it's okay for now